frontend/.eslintrc.cjs

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   extends: [
     "eslint:recommended",
-    "plugin:@typescript-eslint/recommended-type-checked",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
I'd prefer to keep the type-checked settings here, if you don't mind.

Reply:
That was for Eslint to ignore the `showSaveFilePicker` property, but it's no longer needed since adding the type `wicg-file-system-access` solved it!